tests/providers/test_adapter.py

```diff
@@ -386,14 +427,110 @@ def test_multiple_registers(self):
         self.assertEqual(braket_circuit, expected_braket_circuit)
 
 
+class TestFromBraket(TestCase):
+    """Test Braket circuit conversion."""
+
+    def test_type_error_on_bad_input(self):
+        """Test raising TypeError if adapter does not receive a Braket Circuit."""
+        circuit = Mock()
+
+        message = f"Cannot convert {type(circuit)} to Qiskit circuit."
+        with pytest.raises(TypeError, match=message):
+            from_braket(circuit)
+
+    def test_standard_gates(self):
+        """
+        Tests braket to qiskit conversion with standard gates.
+        """
+        braket_circuit = Circuit().h(0)
+        qiskit_circuit = from_braket(braket_circuit)
```

Comment:
Why not `to_qiskit`, or `braket_to_qiskit`? `from_braket` doesn't give any information about the output, which is usually the most important bit. If I see something like `circuit2 = <X>(circuit1)`, I mostly need to know the type of `circuit2` to keep reading